README.md

```diff
@@ -36,34 +36,35 @@ Instead, you'll create your own new node project, and install the oss-mariner pa
 You'll also need a GitHub token and a config file. (Keep reading for more info on these.)
 
 
-### Step-by-step Instructions for running Mariner
+### Step-by-step instructions for creating a project that uses Mariner
 
 1. Create a new project folder and use `npm init` to make it a node project.
 1. Copy the contents of `runFasterCode.ts` into `index.js`.
     - <https://github.com/indeedeng/Mariner/blob/master/examples/runFasterCode.ts>
 1. In `index.js` comment out the existing line that imports mariner.
 1. Also in `index.js`, uncomment the line saying how mariner would normally be imported.
-1. Next, create a folder named `examples` and create two new files inside of it: `config.json` and `exampleData.json`. You can copy the contents of our examples into those new files or you can use the examples as a template for your own data and config choices. The `exampleData.json` should contain the repos that you're interested in getting issues from. You can use a full url or simply `orgname/reponame` for each repo. More info on the config.json can be found [here](https://github.com/indeedeng/Mariner/blob/master/README.md#config.json-format)
-    - <https://github.com/indeedeng/Mariner/blob/master/examples/config.json>
+1. Next, create a folder named `examples` and create two new files inside of it: `exampleData.json` and `config.json`. You can copy the contents of our examples into those new files or you can use the examples as a template for your own data and config choices. The `exampleData.json` should contain the repos that you're interested in getting issues from. For more info on the format of this file, look at[the Input File Format section](https://github.com/indeedeng/Mariner#input-file-format). More info on config.json can also be found [below](https://github.com/indeedeng/Mariner/blob/master/README.md#config.json-format) and the example files can be found here:
```

Comment:
Not a necessary change, but if you make further edits in this file, it might be nice to split this paragraph up into multiple lines, to make future diffs cleaner. I believe that's possible in markdown.